src/librustdoc/passes/rustc_checks.rs

```diff
@@ -0,0 +1,14 @@
+use crate::clean;
+use crate::core::DocContext;
+use crate::passes::Pass;
+
+crate const RUSTC_CHECKS: Pass = Pass {
+    name: "rustc-checks",
+    run: run_rustc_checks,
+    description: "counts the number of items with and without documentation",
+};
+
+crate fn run_rustc_checks(krate: clean::Crate, ctx: &mut DocContext<'_>) -> clean::Crate {
+    rustc_passes::stability::check_unused_or_stable_features(ctx.tcx);
```

Comment:
This seems way overkill for a single line of code (and also not great from the behavior difference - I don't think people should be able to turn this off with --no-default-passes). Can you move it to core directly instead?

Reply:
I just thought we could potentially expand this in the future. But I can move it in `core` directly if you prefer.

Reply:
I think anything like this in the future should also be in core.

Reply:
Fine by me! Gonna move this function then (not calling it directly I will!).